lib/cylc/task_proxy.py

```diff
@@ -234,7 +233,7 @@ def __init__(
         self.poll_timer = None
         self.timeout = None
         self.try_timers = {}
-        self.non_unique_events = Counter()
+        self.non_unique_events = {}
```

Comment:
Worth a comment to remind us to replace with `Counter()` at Python 3?

Reply:
Done.